<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sqlite Explorer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView  
{
selection-background-color: red;

}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layoutCombo">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tables:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTables"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDatabase"/>
   <addaction name="actionCloseDatabase"/>
  </widget>
  <action name="actionOpenDatabase">
   <property name="icon">
    <iconset resource="../res/Resources.qrc">
     <normaloff>:/Icons/32/icons/32/OpenDatabase.png</normaloff>:/Icons/32/icons/32/OpenDatabase.png</iconset>
   </property>
   <property name="text">
    <string>Open database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseDatabase">
   <property name="text">
    <string>Close Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
